<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandRConfigBase</class>
 <widget class="QWidget" name="RandRConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Configuration (X11 Resize, Rotate and Reflect)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="unifyOutputs">
         <property name="text">
          <string>Unify outputs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="outputList" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Primary output:</string>
           </property>
           <property name="buddy">
            <cstring>primaryDisplayBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="primaryDisplayBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="identifyOutputsButton">
         <property name="text">
          <string>Identify Outputs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsDefaultButton">
         <property name="text">
          <string>Save as Default</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="screenView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>unifyOutputs</tabstop>
  <tabstop>primaryDisplayBox</tabstop>
  <tabstop>identifyOutputsButton</tabstop>
  <tabstop>saveAsDefaultButton</tabstop>
  <tabstop>screenView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
